### DISCUSSION TITLE
[Fixed] Can't fin variable: I

### DISCUSSION THREAD

**Discussion by Nevu** [ingrammicro/puzzle-publisher#68] [General]
Hi,

a friend of mine likes to use PUZZLE. Sketch Version is 79.1, and OS is Monterey 12.0.1.
When she tries to export an artboard via Puzzle she gets the following error-message:

<img width="273" alt="error" src="https://user-images.githubusercontent.com/2456068/141989722-a4f36ba6-d498-4ea3-865b-b97ad385296f.png">

The sketch-file is stored local at the desktop, not in Bo, Dropbox or anything else.
Any ideas?

Regards

**Comment by Nevu** [comment]
We tried different files. Every time the same. But here is on elf „her“ file
[test.sketch.zip](https://github.com/ingrammicro/puzzle-publisher/files/7546876/test.sketch.zip)
s:

**Reply by MaxBazarov** [reply]
Thanks for the report. Checking..

**Reply by MaxBazarov** [reply]
> We tried different files. Every time the same. But here is on elf „her“ file
> [test.sketch.zip](https://github.com/ingrammicro/puzzle-publisher/files/7546876/test.sketch.zip)

It works for me. I tried to reproduce using new user context - it works fine.
Could you enable Debug log in Configure Plugin, reproduce the issue and send me a log file lines?
~/Library/Logs/com.bohemiancoding.sketch3/Plugin\ Log.log

**Comment by MaxBazarov** [comment]
@Nevu I think that I found an error. Releases 16.11.4 with hot fix.

BTW, how do you your friend install Puzzle Publisher? The error was not pushed into .zip, so the official way to deliver update should not bring such errors to users.

**Reply by Nevu** [reply]
I'll ask her about the installation :) 
Thankyou for the quick fix!
